hub/management/commands/import_council_emissions_totals.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+from mysoc_dataset import get_dataset_url
+
+from hub.import_utils import add_gss_codes, filter_authority_type
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+declare_map = {
+    "Y": "Yes",
+    "N": "No",
+}
+
+
+class Command(MultipleAreaTypesMixin, BaseImportFromDataFrameCommand):
+    cons_row = "gss_code"
+    message = "Importing council emissions data"
+    uses_gss = True
+    do_not_convert = True
+
+    area_types = ["STC", "DIS"]
+
+    defaults = {
+        "label": "Total Authority Emissions for 2020 (ktCO2)",
```

Comment:
I’d word this as “Total emissions (ktCO2)” – we don’t put dates in any other dataset labels, and we already mention the 2020 date in the `description` and `release_date`.